include/seqan3/io/sequence_file/input.hpp

```diff
@@ -845,7 +845,7 @@ struct tuple_size<seqan3::sequence_file_input<traits_type, selected_field_ids, v
 //!\brief std::tuple_element overload for column-like access. [metafunction specialisation for seqan3::sequence_file_input]
 template <size_t                                             elem_no,
           seqan3::SequenceFileInputTraits                    traits_type,
-          seqan3::detail::fields_concept                     selected_field_ids,
+          seqan3::detail::Fields                             selected_field_ids,
           seqan3::detail::TypeListOfSequenceFileInputFormats valid_formats,
```

Comment:
@smehringer We decided we wanted to get rid of the concepts here and instead perform a static_assert inside. Can you discuss this with @MitraDarja after this PR? Then maybe we can get rid of the detail:: concepts in our public API asap.

Reply:
Originally yes, because we thought the compile time will drop considerably but I tried this for sequence file input and the compile time of the unit test did not change much :( So I don't know if it is worth the effort? I'll add a card for the next strategy meeting.